packages/toolpad-app/src/canvas/getPageViewState.ts

```diff
@@ -94,6 +94,11 @@ export function getNodesViewInfo(rootElm: HTMLElement): {
           const parentId: NodeId = fiber.memoizedProps.parentId as NodeId;
           const nodeSlots = nodes[parentId]?.slots;
 
+          // If there are 2 slots for the same prop name, consider only the first one
```

Comment:
Which scenario would that be?

Reply:
Without this exception, every item rendered by the iterator would have its own slot where you can build layout - and any changes made in any of those slots would always apply to every item. 
So this was a quick way to make it so that in cases like this it's only possible to build a layout in the first slot, using it as a template.